src/platform/sdl/main.c

```diff
@@ -321,3 +337,61 @@ static void mSDLDeinit(struct mSDLRenderer* renderer) {
 
 	SDL_Quit();
 }
+
+void setLogger(struct mCore* core) {
+	int fakeBool = 0;
+	bool logToStdout = true;
+	bool logToFile = false;
+
+	if (mCoreConfigGetIntValue(&core->config, "logToStdout", &fakeBool)) {
+		logToStdout = fakeBool;
+	}
+	if (mCoreConfigGetIntValue(&core->config, "logToFile", &fakeBool)) {
+		logToFile = fakeBool;
+	}
+	const char* logFile = mCoreConfigGetValue(&core->config, "logFile");
+
+	// Assign basic static variables
+	_logToStdout = logToStdout;
+	_logFile = NULL;
```

Comment:
These lines are no longer necessary with the static initialization above.

Reply:
Fixed in https://github.com/mgba-emu/mgba/pull/2345/commits/fac41951f3984e0395765df6c02f0fb02c0ab800 .